.ado/publish.yml

```diff
@@ -5,38 +5,156 @@
 variables:
   Ver.Major: 0
   Ver.Minor: 0
-  Ver.Patch: 0
+  Ver.Patch: 4
 
 name: $(BuildId)
 
 trigger: none
 pr: none
 
 jobs:
-- job: "Publish"
+- job: "Build"
+  strategy:
+    matrix:
+      #linux:                           # 2023.06.30 ESRP Publishing of Py packs is not supported for linux_x86_64.
+      #  imageName: 'ubuntu-latest'     # TODO(kuzminrobin): Manilinux.
```

Comment:
I would like to resolve this TODO in a separate PR.

Reply:
It should be 'manylinux'. Also, being that Linux is a key platform for our target audience (either using Linux directly, or via WSL), it would be good to get this working at the same time if possible.

Reply:
I consider this item resolved and ARM64 for Mac and Win will be added in a separate PR.